util/travis/clangtidy.sh

```diff
@@ -5,6 +5,8 @@ needs_compile || exit 0
 
 if hash clang-tidy-5.0 2>/dev/null; then
 	CLANG_TIDY=clang-tidy-5.0
```

Comment:
should this be removed? (same applies to ` util/travis/lint.sh`)

Reply:
exact